napari/plugins/_plugin_manager.py

```diff
@@ -97,9 +95,6 @@ def __init__(self) -> None:
         self._function_widgets: Dict[str, Dict[str, Callable[..., Any]]] = {}
         self._theme_data: Dict[str, Dict[str, Theme]] = {}
 
-        if sys.platform.startswith('linux') and running_as_bundled_app():
-            sys.path.append(user_site_packages())
-
     def _initialize(self):
```

Comment:
Bye to this hack :D `user_site_packages()` was something like `{sys.prefix}/plugins/lib/pythonX.Y/site-packages` in Briefcase, I think, but nowhere else!